packages/gestalt/src/Checkbox.js

```diff
@@ -100,7 +107,7 @@ export default function Checkbox({
         marginRight={-1}
       >
         <Label htmlFor={id}>
-          <Box paddingX={1} position="relative">
```

Comment:
@AlbertCarreras I do think we need `position="relative"` for browser compatibility. Could you try this change out in:
* Safari
* Chrome
* IE11

by:
* validating the click target is still correct
* validating layout looks good
* it's accessible (you can hit space to toggle the checkbox)

An alternative would be to set an explicit size on `input`

Reply:
ALSO removing position='relative' allows Flyout to get positioned correctly. With it, 
![image](https://user-images.githubusercontent.com/10593890/87819698-863f6900-c821-11ea-807e-2183f64f6437.png)